appinfo/app.php

```diff
@@ -29,5 +30,5 @@
 
     // the title of your application. This will be used in the
     // navigation or on the settings page of your app
-    'name' => \OC_L10N::get('maps')->t('Maps')
+    'name' => $l->t('Maps')
 ));
```

Comment:
While we are on it, let's remove all the deprecated function calls:
```PHP
[...]

\OC::$server->getNavigationManager()->add(array(
    // the string under which your app will be referenced in owncloud
    'id' => 'maps',

    // sorting weight for the navigation. The higher the number, the higher
    // will it be listed in the navigation
    'order' => 10,

    // the route that will be shown on startup
    'href' => \OC::$server->getURLGenerator()->linkToRoute('maps.page.index'),

    // the icon that will be shown in the navigation
    // this file needs to exist in img/
    'icon' => \OC::$server->getURLGenerator()->imagePath('maps', 'maps.svg'),

    // the title of your application. This will be used in the
    // navigation or on the settings page of your app
    'name' => \OC::$server->getL10N('maps')->t('Maps')
));
```